bokehjs/src/coffee/common/models.coffee

```diff
@@ -151,6 +151,7 @@ module.exports = {
   SelectTool:               require '../models/tools/gestures/select_tool'
   ResizeTool:               require '../models/tools/gestures/resize_tool'
   TapTool:                  require '../models/tools/gestures/tap_tool'
+  WheelPanTool:            require '../models/tools/gestures/wheel_pan_tool'
   WheelZoomTool:            require '../models/tools/gestures/wheel_zoom_tool'
 
```

Comment:
I'm not sure how whitespace-aware coffeescript is, but it still seems like one more space here would help. :-)